Patching add-teams.md (#6227)
Kong/docs.konghq.com: app/konnect/dev-portal/access-and-approval/add-teams.md

---
title: Add Developer Teams from Identity Providers
content_type: how-to
---

> In the {{site.konnect_short_name}} Dev Portal, you can map existing developer teams from a third-party identity provider (IdP) and map their permissions to elements in {{site.konnect_short_name}}. Instead of needing to manually copy over each team of developers, with IdP Mapped Teams, developers, as well as their respective permissions, will now be automatically mapped to Konnect.

Reviewer: ```suggestion
In the {{site.konnect_short_name}} Dev Portal, you can map existing developer teams from a third-party identity provider (IdP) and their permissions to elements in {{site.konnect_short_name}}. With teams mapped from an IdP, the developers and permissions are mapped automatically in {{site.konnect_short_name}} so you don't have to manually copy over each team of developers.
```
